app/controllers/projects_controller.rb

```diff
@@ -123,7 +123,11 @@ def index
               project_hash["owner"] = (project_hash["uploaders"] || '').split('::')[0]
               project_hash["editable"] = current_user.admin? || project_hash["editable"] == 1
               project_hash.delete("uploaders")
-              ["locations", "hosts", "tissues"].each { |k| project_hash[k] = (project_hash[k] || '').split('::') }
+              ["locations", "hosts", "sample_types"].each { |k| project_hash[k] = (project_hash[k] || '').split('::') }
+              # Return as "tissue" for legacy compatibility. It's too hard to
+              # rename all JS instances of "tissue".
```

Comment:
should we have a TODO/JIRA for this?

Reply:
For a minor code quality issue like this, I would leave it to the next person who works with this code to escalate to a ticket for team prioritization, according to their judgement. In response to the new team convention that every TODO should have a ticket, I'm starting to add mere comments when I identify minor issues. As in this case, just knowing that someone tried to fix something is valuable.